tox.ini

```diff
@@ -0,0 +1,62 @@
+[tox]
+envlist = py36-tests
```

Comment:
```suggestion
envlist = tests
```

(See comment above about not duplicating `py36-`), although I think this may be a bug in the cookiecutter

Reply:
This is a cookie cutter thing yes.